.gitlab-ci.yml

```diff
@@ -0,0 +1,26 @@
+# Official docker image.
+image: docker:latest
+
+services:
+  - docker:dind
+
+before_script:
+  - docker login -u $DOCKER_CI_USER -p $DOCKER_CI_PASS
+
+build-tag:
+  stage: build
+  script:
+    # - docker run --rm -i hadolint/hadolint < Dockerfile
+    - docker build -t "flywheel/c-pac:$CI_COMMIT_REF_NAME" .
+    - docker push "$flywheel/c-pac:$CI_COMMIT_REF_NAME"
+  only:
+    - tags
+
+build:
+  stage: build
+  script:
+    # - docker run --rm -i hadolint/hadolint < Dockerfile
+    - docker build -t "flywheel/c-pac:latest" .
+  except:
+    - tags
```

Comment:
Would it make sense to have a build stage and then a deploy stage that is for only tags? Or does it not matter that much?

Reply:
We could, but we would need to [push/pull the container between stages](https://docs.gitlab.com/ee/ci/docker/using_docker_build.html#container-registry-examples). I think this build is simple enough that we can duplicate the couple of lines, and will be efficient in that it only runs one job for each push/tag.